### ISSUE TITLE
Addon failed to start (s6-rc: fatal: timed out in pgadmin4.sh): Increase s6 timout.

### ISSUE THREAD

**Issue by akanouras** [Expaso/hassos-addon-pgadmin4#7]
HA port: 8123
Nginx SSL port: 54123
Internal/External URL is not configured in HA settings.

Is there a quick way to get to that `/run/uncaught-logs/current` file?

System log:
```
Jan 11 14:50:45 homeassistant hassio_supervisor[389]: 23-01-11 16:50:45 INFO (SyncWorker_1) [supervisor.docker.addon] Starting Docker add-on husselhans/hassos-addon-pgadmin4-amd64 with version 2.1.1
Jan 11 14:50:46 homeassistant addon_77b2833f_pgadmin4[389]: s6-rc: info: service s6rc-oneshot-runner: starting
Jan 11 14:50:46 homeassistant addon_77b2833f_pgadmin4[389]: s6-rc: info: service s6rc-oneshot-runner successfully started
Jan 11 14:50:46 homeassistant addon_77b2833f_pgadmin4[389]: s6-rc: info: service fix-attrs: starting
Jan 11 14:50:46 homeassistant addon_77b2833f_pgadmin4[389]: s6-rc: info: service fix-attrs successfully started
Jan 11 14:50:46 homeassistant addon_77b2833f_pgadmin4[389]: s6-rc: info: service legacy-cont-init: starting
Jan 11 14:50:46 homeassistant addon_77b2833f_pgadmin4[389]: cont-init: info: running /etc/cont-init.d/00-banner.sh
Jan 11 14:50:46 homeassistant addon_77b2833f_pgadmin4[389]: 
Jan 11 14:50:46 homeassistant addon_77b2833f_pgadmin4[389]: -----------------------------------------------------------
Jan 11 14:50:47 homeassistant addon_77b2833f_pgadmin4[389]:  Add-on: pgAdmin4
Jan 11 14:50:47 homeassistant addon_77b2833f_pgadmin4[389]:  A PostgreSQL Management and Query tool
Jan 11 14:50:47 homeassistant addon_77b2833f_pgadmin4[389]: -----------------------------------------------------------
Jan 11 14:50:48 homeassistant addon_77b2833f_pgadmin4[389]:  Add-on version: 2.1.1
Jan 11 14:50:48 homeassistant addon_77b2833f_pgadmin4[389]:  You are running the latest version of this add-on.
Jan 11 14:50:49 homeassistant addon_77b2833f_pgadmin4[389]:  System: Home Assistant OS 9.4  (amd64 / generic-x86-64)
Jan 11 14:50:50 homeassistant addon_77b2833f_pgadmin4[389]:  Home Assistant Core: 2023.1.2
Jan 11 14:50:50 homeassistant addon_77b2833f_pgadmin4[389]:  Home Assistant Supervisor: 2022.12.1
Jan 11 14:50:50 homeassistant addon_77b2833f_pgadmin4[389]: -----------------------------------------------------------
Jan 11 14:50:50 homeassistant addon_77b2833f_pgadmin4[389]:  Please, share the above information when looking for help
Jan 11 14:50:50 homeassistant addon_77b2833f_pgadmin4[389]:  or support in, e.g., GitHub, forums or the Discord chat.
Jan 11 14:50:50 homeassistant addon_77b2833f_pgadmin4[389]: -----------------------------------------------------------
Jan 11 14:50:50 homeassistant addon_77b2833f_pgadmin4[389]: cont-init: info: /etc/cont-init.d/00-banner.sh exited 0
Jan 11 14:50:50 homeassistant addon_77b2833f_pgadmin4[389]: cont-init: info: running /etc/cont-init.d/01-log-level.sh
Jan 11 14:50:51 homeassistant addon_77b2833f_pgadmin4[389]: cont-init: info: /etc/cont-init.d/01-log-level.sh exited 0
Jan 11 14:50:51 homeassistant addon_77b2833f_pgadmin4[389]: cont-init: info: running /etc/cont-init.d/nginx.sh
Jan 11 14:50:52 homeassistant hassio_supervisor[389]: 23-01-11 16:50:52 INFO (MainThread) [supervisor.api.middleware.security] /dns/info access from 77b2833f_pgadmin4
Jan 11 14:50:53 homeassistant addon_77b2833f_pgadmin4[389]: cont-init: info: /etc/cont-init.d/nginx.sh exited 0
Jan 11 14:50:53 homeassistant addon_77b2833f_pgadmin4[389]: cont-init: info: running /etc/cont-init.d/pgadmin4.sh

Jan 11 14:51:36 homeassistant addon_77b2833f_pgadmin4[389]: s6-rc: fatal: timed out
Jan 11 14:51:36 homeassistant addon_77b2833f_pgadmin4[389]: /run/s6/basedir/scripts/rc.init: warning: s6-rc failed to properly bring all the services up! Check your logs (in /run/uncaught-logs/current if you have in-container logging) for more information.
Jan 11 14:51:36 homeassistant addon_77b2833f_pgadmin4[389]: prog: fatal: stopping the container.
Jan 11 14:51:36 homeassistant addon_77b2833f_pgadmin4[389]: s6-sudoc: fatal: unable to get exit status from server: Operation timed out
Jan 11 14:51:36 homeassistant addon_77b2833f_pgadmin4[389]: s6-rc: info: service fix-attrs: stopping
Jan 11 14:51:36 homeassistant addon_77b2833f_pgadmin4[389]: s6-rc: info: service fix-attrs successfully stopped
Jan 11 14:51:36 homeassistant addon_77b2833f_pgadmin4[389]: s6-rc: info: service s6rc-oneshot-runner: stopping
Jan 11 14:51:36 homeassistant addon_77b2833f_pgadmin4[389]: s6-rc: info: service s6rc-oneshot-runner successfully stopped
Jan 11 14:51:40 homeassistant systemd[1]: docker-3c13887286a4f9c3709cb8be1e67f947bd9b51aa68bed3857cdac70dd3ea3f3d.scope: Deactivated successfully.
Jan 11 14:51:40 homeassistant systemd[1]: docker-3c13887286a4f9c3709cb8be1e67f947bd9b51aa68bed3857cdac70dd3ea3f3d.scope: Consumed 50.315s CPU time.
```
Addon configuration:
```yaml
ssl: false
certfile: fullchain.pem
keyfile: privkey.pem
system_packages: []
init_commands: []
leave_front_door_open: false
```
Web interface (Not required for Ingress): _(empty)_
Web interface SSL (Not required for Ingress): _(empty)_

**Comment by charlesbaynham** [comment]
For me this occurred on the first attempted start but, when I tried again, it worked the second time.

**Comment by expaso** [comment]
If an operation time-out. I think the hardware is not beefy enough, but I will increase the timeout of the addons nonetheles.

**Comment by expaso** [comment]
https://github.com/Expaso/hassos-addons/pull/37

A new version was released 3.0.1, updating  pgAdmin to 8.2, and increased the startup times.
Let me know of any issues by opening a ticket here if needed.

Enjoy!
